README.md

```diff
@@ -1 +1,35 @@
-# HTTP_Toolkit
\ No newline at end of file
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/yx1xgac1ygxefwna/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/http_toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/HTTP_Toolkit/HTTP_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build?definitionId=67&branchName=master)
+
+# HTTP_Toolkit
+
+This toolkit allows interoperability between the BHoM and HTTPClient operations. It is generally used in combination with another toolkit, such as the CarbonQueryDatabase_Toolkit, in order to interact with a specific HTTP-based platform.
```

Comment:
Just to avoid having to update this ReadMe again in the future, I feel we should refrain from making reference to specific toolkits as the 'such as'. Just leave as `It is generally used in combination with other toolkits`.

Reply:
Fair point, updated.